README.md

```diff
@@ -37,6 +39,17 @@ Livros recomendados pelos participantes das comunidades DevAraraquara / OpenSanc
 [Baixar (em português)](http://why.carlosbrando.com/)
 * **Comentários:**
   * _Why's (Poignant) Guide to Ruby_ é um livro para iniciantes em Ruby que surpreende a cada página, seja pela quantidade exorbitante de histórias, bacon ou quadrinhos de raposa.
+  * 
+  
+## Cloud Computing
+
+### <a name="Amazon Web Services in Action"></a>Amazon Web Services in Action
```

Comment:
This need to be the same value of the anchor.
So, your anchor is: `#amazon-web-services-in-action`, and this need to be
<a name="amazon-web-services-in-action"></a>

This text don't have special chars ( ` ' ", so, you can remove and keep only:
### Amazon Web Services in Action